keyboards/satt/vision/info.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
info.json renders like this in Configurator:
![image](https://user-images.githubusercontent.com/4781841/90261029-b7966d80-de8f-11ea-8262-10d21e825f55.png)

Reply:
Sorry, I forgot to check this. Fixed in 613aab3.
Although I had rotation set to 0 for all keys, the rotation center was non-zero (13.5, 3.5) which resulted in an error when converting XY-coordinates from KLE-raw with "Convert KLE raw to QMK info.json".